doc/source/cd/background.md

```diff
@@ -0,0 +1,869 @@
+# Background
+
+```{contents}
+:depth: 2
+:local: true
+```
+
+## 1. What is drift?
+
+Although powerful, modern machine learning models can be sensitive.
+Seemingly subtle changes in a data distribution can destroy the
+performance of otherwise state-of-the art models, which can be
+especially problematic when ML models are deployed in production.
+Typically, ML models are tested on held out data in order to estimate
+their future performance. Crucially, this assumes that the *process*
+underlying the input data $\mathbf{X}$ and output data
+$\mathbf{Y}$ remains constant.
+
+*Drift* is said to occur when the process underlying $\mathbf{X}$
+and $\mathbf{Y}$ at test time differs from the process that
+generated the training data. In this case, we can no longer expect the
+model’s performance on test data to match that observed on held out
+training data. At test time we always observe features $\mathbf{X}$,
+and the *ground truth* then refers to a corresponding label $\mathbf{Y}$. 
+If ground truths are available at test time,
+*supervised drift detection* can be performed, with the model’s
+predictive performance monitored directly. However, in many scenarios,
+such as the binary classification example below, ground truths are not
+available and *unsupervised drift detection* methods are required.
+
+```{image} images/drift_deployment.png
+:align: center
+:alt: Drift in deployment
+```
+
+To explore the different types of drift, consider the common scenario
+where we deploy a model $f: \boldsymbol{x} \mapsto y$ on input data 
+$\mathbf{X}$ and output data $\mathbf{Y}$, jointly distributed according to
+$P(\mathbf{X},\mathbf{Y})$. The model is trained on training data drawn from
+a distribution $P_{ref}(\mathbf{X},\mathbf{Y})$. *Drift* is said
+to have occurred when
+$P(\mathbf{X},\mathbf{Y}) \ne P_{ref}(\mathbf{X},\mathbf{Y})$.
+Writing the joint distribution as
+
+$$
+P(\mathbf{X},\mathbf{Y}) = P(\mathbf{Y}|\mathbf{X})P(\mathbf{X}) = P(\mathbf{X}|\mathbf{Y})P(\mathbf{Y}),
+$$
+
+we can classify drift under a number of types:
+
+- **Covariate drift**: Also referred to as input drift, this occurs
+  when the distribution of the input data has shifted
+  $P(\mathbf{X}) \ne P_{ref}(\mathbf{X})$, whilst
+  $P(\mathbf{Y}|\mathbf{X})$ =
+  $P_{ref}(\mathbf{Y}|\mathbf{X})$. The model may now be
+  extrapolating, potentially resulting in unreliable predictions.
+- **Prior drift**: Also referred to as label drift, this occurs when
+  the distribution of the outputs has shifted
+  $P(\mathbf{Y}) \ne P_{ref}(\mathbf{Y})$, whilst
+  $P(\mathbf{X}|\mathbf{Y})=P_{ref}(\mathbf{X}|\mathbf{Y})$. The
+  model could still be accurate, but accuracy metrics might be
+  affected. Additionally, it is common for the optimal decision 
+  boundary to change under prior drift.
```

Comment:
Seems to be three clauses here making the same point. Could we not compress them to something along the lines of "the change could affect the model decision boundary and performance metrics"